builtInFunctions/creator.go

```diff
@@ -129,7 +129,7 @@ func (b *builtInFuncCreator) CreateBuiltInFunctionContainer() error {
 		return err
 	}
 
-	newFunc, err = NewSaveUserNameFunc(b.gasConfig.BuiltInCost.SaveUserName, b.mapDNSAddresses, b.enableUserNameChange)
+	newFunc, err = NewSaveUserNameFunc(b.gasConfig.BuiltInCost.SaveUserName, b.mapDNSAddresses)
```

Comment:
[question]

Why does this func does not require `b.enableEpochsHandler` anymore?

Reply:
it was not used.